examples/frame/src/frame/index.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { root, useState } from "@lynx-js/react";
+
+interface MyInitData {
+  id: number;
+  name: string;
+  showDetail: boolean;
+}
+
+interface MyGlobalProps {
+  message: string;
+  status: number;
+  loaded: boolean;
+}
+
+const App = () => {
+  const [bindLoadEvent, setBindLoadEvent] = useState({
+    url: "",
+    statusCode: 0,
+    statusMessage: "",
+  });
+  const [initData, setInitData] = useState<MyInitData>({
+    id: 123,
+    name: "Alice",
+    showDetail: true,
+  });
+  const [globalProps, setGlobalProps] = useState<MyGlobalProps>({
+    message: "hello globalProps!",
+    status: 42,
+    loaded: true,
+  });
+  const handleLoad = (e) => {
+    setBindLoadEvent(e.detail);
+  };
+
+  const frameUrl = "https://tosv-sg.tiktok-row.org/obj/lynx-artifacts-oss-sg/lynx-website/assets/in_frame.lynx.bundle";
```

Comment:
I don't think it's a good idea to use a CDN URL like this. Maybe we could use the `process.env.ASSET_PREFIX`:

```js
const frameUrl = `${process.env.ASSET_PREFIX}/in_frame.lynx.bundle`;
```